vtex/omnishipping
# CHUI-77: Fix mandatory invoice address

#### What is the purpose of this pull request?


This PR fixes a bug, that happens when the invoice is active, which would allow the user to proceed to the payment step without having inserted a valid invoice address.

Now, the "go to payment" button is only present when a proper invoice address is inserted.

#### How should this be manually tested?

- Use this [workspace](https://jeff--vtexgame1invoice.myvtex.com/checkout/cart/add/?sku=297&qty=1&seller=1&sc=1)
- Proceed to the Omnishipping
- Search for pickup points and use `22250-040` as the postal code
- Pick `Loja em Copacabana`
- Click on "Preencher endereço de nota fiscal"
- You shouldn't see the "go to payment" button until the invoice address is properly inserted.

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12574426/85404514-6a53ea80-b535-11ea-948d-078f4a394d56.png)


#### Types of changes
